source-postgres/replication.go

```diff
@@ -69,6 +69,8 @@ func (db *postgresDatabase) ReplicationStream(ctx context.Context, startCursor s
 		// completely unable to produce a fatal error, and double-checking to make extra sure we don't
 		// accidentally dereference a null pointer doesn't really hurt anything.
 		logrus.WithField("slot", slot).Warn("replication slot has no metadata (and probably doesn't exist?)")
+	} else if slotInfo.Active {
+		logrus.WithField("slot", slot).Warn("replication slot is already active (is another capture already running against this database?)")
```

Comment:
should we be error-ing out when this happens? A warning will be easy to ignore and I imagine this is going to give incorrect results.

Reply:
This whole section of logic is deliberately only logging warnings for now. Eventually all of these warnings will be upgraded to errors once I'm confident they're working correctly in production.

Also in the specific case of this "is the slot active?" check, we know that the attempt to actually start replication a few lines further down in this function will fail anyway.